spec/plane_spec.rb

```diff
@@ -17,14 +18,33 @@
 
 describe Plane do
 
-  xit 'is flying when created'
+subject {Plane.new}
+let(:airport) { Airport.new }
 
-  xit 'can land'
+  before(:each) do
+      allow(airport).to receive(:can_land).and_return true
+      allow(airport).to receive(:can_take_off).and_return true
+    end
 
-  xit 'is landed after landing'
+  it 'is flying when created' do
+    expect(subject.status).to eq 'Flying'
+  end
 
-  xit 'can take off'
+  it 'can land' do
+    expect(subject.landed).to eq 'Landed'
+  end
 
-  xit 'is flying after take off'
+  it 'is landed after landing' do
+    airport.land_plane(subject)
+    expect(subject.status).to eq 'Landed'
+  end
 
+  it 'can take off' do
+    expect(airport.take_off_plane(subject)).to eq 'Took off!'
+    end
```

Comment:
Lint/BlockAlignment: `end` at 44, 4 is not aligned with `it 'can take off' do` at 42, 2